test/unit/tasks/fixes_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+module Tasks
+  class FixesTest < ActiveSupport::TestCase
+    test 'regenerate JPEG logos :invoice style' do
+      profile = FactoryBot.create(:profile)
+      profile.logo.post_processing = false
+      file_fixture("hypnotoad.jpg").open("rb") { |io| profile.logo = io }
+      profile.save!
+
+      tmpfile = Tempfile.new(["logo-", ".jpg"])
+      assert_raises(Errno::ENOENT) { profile.logo.copy_to_local_file :invoice, tmpfile.path }
+
+      execute_rake_task 'fixes.rake', 'fixes:regenerate_jpeg_invoice_logo'
+      profile.logo.copy_to_local_file :invoice, tmpfile.path
+    end
```

Comment:
I would add a new test to ensure it's not reprocessing gifs and pngs

Reply:
done